nixpkgs_review/cli/__init__.py

```diff
@@ -138,6 +138,11 @@ def parse_args(command: str, args: List[str]) -> argparse.Namespace:
             type=regex_type,
             help="Regular expression that package attributes have to match (can be passed multiple times)",
         ),
+        CommonFlag(
+            "--batch",
```

Comment:
Please move this to common_flags and also make it work for the `rev` command.
Also rename it to `--no-shell`.

Reply:
Tested working for all 3 commands; it's also already in common_flags.